atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -179,7 +179,7 @@ def add_as_an_alias(set_to, set_from):
 
         for old, new in self.aliases.items():
             if old not in all_labels:
-                applied_alias = not_applied.append(old)
+                not_applied.append(old)
```

Comment:
This also changes the logic, but I'm pretty sure the original logic was a bug and it's more correct now (and it's only for a log message anyway)

Reply:
Honestly I'm confused with that logic there, why is there even function that always returns `True` :-|

Reply:
Right? :smile: 

I was going to refactor that - I'm pretty sure `applied_alias` is useless now and the condition could just be `if alias_labels and not_applied`. But then I got lost and gave up, better to leave this skeleton in the closet :smile: